jbmc/src/java_bytecode/java_bytecode_parser.cpp

```diff
@@ -1881,6 +1881,12 @@ void java_bytecode_parsert::parse_local_variable_type_table(methodt &method)
   }
 }
 
+/// Translate the lambda method reference kind in a class file into the kind
+/// of handling the method requires. invokestatic/special translate into direct
+/// method calls; invokevirtual/interface translate into virtual dispatch,
+/// newinvokespecial translates into a special instantiate-and-construct
+/// sequence. The field-manipulation reference kinds appear never to happen in
+/// reality and don't have syntax in the Java language.
```

Comment:
I think this info would be good to put nearer the block that handles this

Reply:
It's now mentioned in both places